.all-contributorsrc

```diff
@@ -0,0 +1,247 @@
+{
+  "projectName": "pmd",
+  "projectOwner": "pmd",
+  "repoType": "github",
+  "repoHost": "https://github.com",
+  "files": [
+    "README.md"
+  ],
+  "imageSize": 100,
+  "commit": true,
+  "commitConvention": "none",
+  "contributors": [
+    {
+      "login": "adangel",
+      "name": "Andreas Dangel",
+      "avatar_url": "https://avatars.githubusercontent.com/u/1573684?v=4",
+      "profile": "https://github.com/adangel",
+      "contributions": [
+        "code",
+        "doc"
+      ]
+    },
+    {
+      "login": "oowekyala",
+      "name": "Clément Fournier",
+      "avatar_url": "https://avatars.githubusercontent.com/u/24524930?v=4",
+      "profile": "https://www.linkedin.com/in/le-clement-fournier/",
```

Comment:
I'd rather my github profile be listed here. Is it possible to edit this file manually?

Reply:
I've changed it